trailblazer/apps/tower/models.py

```diff
@@ -64,8 +64,9 @@ class Config:
         validate_all = True
 
     @validator("duration")
-    def set_duration(cls, duration) -> str:
-        return duration or 0
+    def set_duration(cls, duration: Optional[int]) -> int:
+        """Convert milliseconds to seconds or return 0 if empty."""
+        return round(duration / 1000) if duration else 0
```

Comment:
```suggestion
        return round(duration / SCALE_TO_MILLISEC) if duration else 0
```
And define the constant perhaps in this module as it is quite specific.
`SCALE_TO_MILLISEC: int = 1_000`